test/src/test/java/org/corfudb/runtime/collections/DiskBackedCorfuClientTest.java

```diff
@@ -0,0 +1,171 @@
+package org.corfudb.runtime.collections;
+
+import com.google.common.reflect.TypeToken;
+import com.google.gson.Gson;
+import io.netty.buffer.ByteBuf;
+import lombok.Builder;
+import lombok.Data;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang.RandomStringUtils;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.exceptions.unrecoverable.UnrecoverableCorfuError;
+import org.corfudb.runtime.view.AbstractViewTest;
+import org.corfudb.util.serializer.ISerializer;
+import org.corfudb.util.serializer.Serializers;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import org.rocksdb.Env;
+import org.rocksdb.Options;
+import org.rocksdb.RocksDB;
+import org.rocksdb.RocksDBException;
+import org.rocksdb.SstFileManager;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.LongStream;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class DiskBackedCorfuClientTest extends AbstractViewTest {
```

Comment:
1. Do we need tests around any kind of failure conditions ?
2. Should we think of having some sort of disk size limitation ?
3. Can we add tests where transactions involve diskbased and memorybased tables ?

Reply:
There is a test for (2). I will address (1) and (3).